docs/resources/azure_data_factories.md

```diff
@@ -0,0 +1,71 @@
+---
+title: About the azure_data_factories Resource
+platform: azure
+---
+
+# azure_data_factories
+
+Use the `azure_data_factories` InSpec audit resource to test properties related to data factories for a resource group or the entire subscription.
+
+## Azure REST API version, endpoint and http client parameters
+
+This resource interacts with api versions supported by the resource provider.
+The `api_version` can be defined as a resource parameter.
+If not provided, the latest version will be used.
+For more information, refer to [`azure_generic_resource`](azure_generic_resource.md).
+
+Unless defined, `azure_cloud` global endpoint, and default values for the http client will be used.
+For more information, refer to the resource pack [README](../../README.md). 
+For api related info : [`Azure Data Factories Docs`](https://docs.microsoft.com/en-us/azure/data-factory/quickstart-create-data-factory-rest-api#create-a-data-factory).
+## Availability
+
+### Installation
+
+This resource is available in the [InSpec Azure resource pack](https://github.com/inspec/inspec-azure). 
+For an example `inspec.yml` file and how to set up your Azure credentials, refer to resource pack [README](../../README.md#Service-Principal).
+
+## Syntax
+
+An `azure_data_factories` resource block returns all Azure data factories, either within a Resource Group (if provided), or within an entire Subscription.
+```ruby
+describe azure_data_factories do
+  #...
+end
+```
+or
+```ruby
+describe azure_data_factories(resource_group: 'my-rg') do
+  #...
+end
+```
+
+## Properties
+
+|Property       | Description                                                                          | Filter Criteria<superscript>*</superscript> |
+|---------------|--------------------------------------------------------------------------------------|-----------------|
+| name           | A list of the unique resource ids.                                                   | `name`            |
```

Comment:
As this is a plural resource, we have to change the field name from name to names. And the plural resource, in the filtertable you have written names

Reply:
Please also add the following fields that you have mentioned in the filter table types, provisioning_states and locations.